src/Microsoft.ML.Transforms/Text/WordTokenizing.cs

```diff
@@ -460,10 +437,29 @@ public WordTokenizingEstimator(IHostEnvironment env, (string outputColumnName, s
         /// </summary>
         /// <param name="env">The environment.</param>
         /// <param name="columns">Pairs of columns to run the tokenization on.</param>
-        public WordTokenizingEstimator(IHostEnvironment env, params WordTokenizingTransformer.ColumnInfo[] columns)
+        internal WordTokenizingEstimator(IHostEnvironment env, params ColumnInfo[] columns)
           : base(Contracts.CheckRef(env, nameof(env)).Register(nameof(WordTokenizingEstimator)), new WordTokenizingTransformer(env, columns))
         {
         }
+        public sealed class ColumnInfo
+        {
+            public readonly string Name;
+            public readonly string InputColumnName;
+            public readonly char[] Separators;
+
+            /// <summary>
+            /// Describes how the transformer handles one column pair.
+            /// </summary>
+            /// <param name="name">Name of the column resulting from the transformation of <paramref name="inputColumnName"/>.</param>
+            /// <param name="inputColumnName">Name of column to transform. If set to <see langword="null"/>, the value of the <paramref name="name"/> will be used as source.</param>
+            /// <param name="separators">Casing text using the rules of the invariant culture. If not specified, space will be used as separator.</param>
+            public ColumnInfo(string name, string inputColumnName = null, char[] separators = null)
+            {
+                Name = name;
+                InputColumnName = inputColumnName ?? name;
+                Separators = separators ?? new[] { ' ' };
+            }
+        }
 
         public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
         {
```

Comment:
/// <summary>
/// Returns the <see cref="SchemaShape"/> of the schema which will be produced by the transformer.
/// Used for schema propagation and verification in a pipeline.
/// </summary>

Reply:
fixed

---
In reply to: [254082141](https://github.com/dotnet/machinelearning/pull/2393#discussion_r254082141) [](ancestors = 254082141)